### ISSUE TITLE
create project logo

### ISSUE THREAD

**Issue by tomkralidis** [pydap/pydap#171]
It would be valuable to create a project logo for general use (website, favicon, etc.).

**Comment by mandyellow** [comment]
Hi, I made a logo for your project. Hope you enjoy! :-)

![ativo 1-100](https://user-images.githubusercontent.com/30158643/46709877-1b103e00-cc1c-11e8-9875-175a44c38044.jpg)

**Comment by 7yl4r** [comment]
My first glance read this as pydag, but I like the overall look.

**Comment by mandyellow** [comment]
WOW, It's true... The letter P looks like a G... Let me try again! :-)
![ativo 3-100](https://user-images.githubusercontent.com/30158643/46742379-4b3bf900-cc7d-11e8-8212-bc2c14ad4fde.jpg)

**Comment by tomkralidis** [comment]
Nice work @mandyellow! Comments:

# Icon
- can we have the icon have rounded corners?
- perhaps just a single 'P' atop an image like https://docs.opendap.org/favicon.ico (same shape, any colour)

**Comment by tomkralidis** [comment]
@7yl4r great work here too! Comments:
- can we make the text Pydap (not pyDAP)?

**Comment by mandyellow** [comment]
@tomkralidis just have a look if I get it right, please

![ativo 4-100](https://user-images.githubusercontent.com/30158643/46754553-235a8e80-cc99-11e8-9486-581f95ef72a0.jpg)

**Comment by tomkralidis** [comment]
@mandyellow exactly, good work!  Looking forward to comments from others.

**Comment by 7yl4r** [comment]
Lol. Sure, here's lowercased.

![pydap](https://user-images.githubusercontent.com/1051390/46755083-b9031780-ccb3-11e8-8794-7892b613aca5.png)

Here's a link to [download the inkscape svg](https://drive.google.com/file/d/14BkcP0WPPqy-B1da9191u7SrrMJZDbeQ/view?usp=sharing) if anyone else wants to play with it.

**Comment by tomkralidis** [comment]
@mandyellow one more example: can you make the log text 'Pydap' instead of 'PYDAP' ?

**Comment by mandyellow** [comment]
Sure :) 
![ativo 6-100](https://user-images.githubusercontent.com/30158643/47126111-70bc9a00-d25d-11e8-8689-0c8c00d12f6a.jpg)

**Comment by jgallagher59701** [comment]
+1

**Comment by tomkralidis** [comment]
@mandyellow based on community input we are going with `pydap` (not `Pydap`) as the canonical spelling of the project.  Would the icon/logo in https://github.com/pydap/pydap/issues/171#issuecomment-428660925 end up being the same?

**Comment by Juanlu001** [comment]
Looking forward to see https://github.com/pydap/pydap/issues/171#issuecomment-428660925 in the website :)

**Comment by captainkirk99** [comment]
Well did we ever get a lower-cased version of the logo? If so, I will add it to the documentation...

**Comment by tomkralidis** [comment]
Sounds good!

**Comment by 7yl4r** [comment]
If alterations are wanted on my file I can dig up my old `.svg` file but I think the consensus was to use @mandyellow 's logo. If ey has the original file in a format I can open with Inkscape then I am happy to modify. I could recreate it too. I will wait a bit to see if ey replies. Ideally we would include the vector file in this repo in addition to the `.jpg`/`.png` exports.
